Packs/SymantecEmailSecurity/Integrations/SymantecEmailSecurity/SymantecEmailSecurity.py

```diff
@@ -0,0 +1,1888 @@
+    first_fetch_time: datetime,
+    max_results: int = MAX_INCIDENTS_TO_FETCH,
+    accepted_severities: list | None = None,
+    feed_type: str = "all",
+    include_delivery: bool = False,
+) -> tuple[dict[str, Any], list[dict[str, Any]]]:
+    """This function will execute each interval (default is 1 minute).
+
+    Args:
+        client (Client): A session to run HTTP requests to the API.
+        last_run (dict[str, Any]): The greatest incident created_time we fetched from last fetch.
+        first_fetch_time (datetime): If last_run is None then fetch all incidents since first_fetch_time.
+        max_results (int, optional): Maximum numbers of incidents per fetch.
+            Defaults to MAX_INCIDENTS_TO_FETCH.
+        feed_type (str, optional): The name of the data feed to retrieve.
+            Available feeds:
+                - all: metadata for all scanned email.
+                - malware: malware-containing email data.
+                - threat-isolation: events from URL and Attachment Isolation.
+                - clicktime: metadata from end-user clicks on rewritten URLs.
+                - anti-spam: spam detection metadata.
+                - ec-reports: contextual information about emails blocked by Anti-Malware service.
+            Defaults to "all".
+        include_delivery (bool, optional): Only relevant to `all` feed.
+            Contains metadata that describes both inbound and outbound email delivery to provide visibility
+            into email tracing, TLS compliance, and routing.
+            Defaults to False
+
+    Returns:
+        tuple[dict[str, Any], list[dict[str, Any]]]:
+            - next_run: This will be last_run in the next fetch-incidents.
+            - incidents: Incidents that will be created in Cortex XSOAR.
+    """
+    if last_fetch := last_run.get("last_fetch"):
+        reset = last_fetch
+    else:
+        reset = first_fetch_time
+
+    last_ids: list[str] = last_run.get("last_ids", [])
+    incidents: list[dict[str, Any]] = []
+
+    demisto.debug(f"New run {last_fetch=}, {last_ids=}.")
+
+    # Reset data feed timer to given start time.
+    client.list_data(
+        feed_type=feed_type,
+        reset=convert_datetime_string(reset),
+    )
+    # Fetch email data feed from start time.
+    raw_response = cast(
+        list[dict[str, Any]],
+        client.list_data(
+            feed_type=feed_type,
+            include="delivery" if include_delivery else None,
+        ),
+    )
+
+    new_ids = []
+    items = raw_response or []
+    demisto.debug(f"Received {len(items)} feed items from the server.")
+
+    for item in items:
+        if not (email_info := item.get("emailInfo")) or not (item_incidents := item.get("incidents")):
+            continue
+
+        incident_id = email_info.get("xMsgRef")
+        new_ids.append(incident_id)
+
+        # Skip incident if occurred time is less than last fetch
+        if incident_id in last_ids:
+            continue
```

Comment:
I fixed the name to `seen_ids` to avoid confusion.
Every ID should be added, if we put this above the append that will cause an ID that has been used to be used again in every 3rd run.